lib/k8s/resource_client.rb

```diff
@@ -200,7 +215,26 @@ def list(labelSelector: nil, fieldSelector: nil, namespace: @namespace)
           'fieldSelector' => selector_query(fieldSelector)
         )
       )
-      process_list(list)
+    end
+
+    # @param labelSelector [nil, String, Hash{String => String}]
+    # @param fieldSelector [nil, String, Hash{String => String}]
+    # @param resourceVersion [nil, String]
+    # @return [Array<resource_class>]
```

Comment:
When does it return?

Reply:
Never, it always raises if watch ends (fixed yardoc).